src/app/components/elements/PageTitle.tsx

```diff
@@ -94,11 +91,7 @@ export const PageTitle = ({currentPageTitle, subTitle, disallowLaTeX, help, clas
             <meta property="og:title" content={currentPageTitle} />
         </Helmet>
         {audienceViews && <AudienceViewer audienceViews={audienceViews} />}
-        {help && !showModal && <React.Fragment>
```

Comment:
This looks wrong... For CS, `showModal` would have been always false. So the help section would have always been visible (providing that `help` is truthy, and similarly the "link" section would have always been false for CS, but here we're now displaying it if there's a `modalId`.

Reply:
Not sure why I did this as it is clearly wrong!! Should be corrected now